docker-compose.yml

```diff
@@ -105,8 +105,8 @@ services:
     build:
       context: ./emailing-service
       dockerfile: emailing-service/Dockerfile
-    #ports:
-     # - "5000:5115" # Used for testing without gateway
+    ports:
+      - "5000:5115" # Used for testing without gateway
```

Comment:
Is this port still open and accessible when upping the system? If so, it should be closed.

Reply:
Yes you are right! I will fix that!